### PR TITLE
Added meilix-generator icon

### LINKED ISSUE #70: Meilix-generator logo in Index
It would be nice if we add the meilix generator logo in README at index page.

### PR DESCRIPTION
**Pull request by yashLadha**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [ ] The unit tests pass locally with my changes (provide a screenshot or link for test) 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Added Logo at index page

#### Changes proposed in this pull request:

-Added Logo 

Fixes #70

### CODE REVIEW

**Comment by yashLadha** [comment]
screenshot from 2017-06-28 12-38-46

**Comment by yashLadha** [comment]
Review @meets2tarun @xeon-zolt

**Comment by meets2tarun** [comment]
@yashLadha Not bad, it looks :sparkles: 
but don't you think it would be much better if that it looks a bit more relative to the webapp.
The image seems to unfit in the webapp.

**Comment by yashLadha** [comment]
So there is a issue with the logo, you are saying @meets2tarun .

**Comment by meets2tarun** [comment]
@yashLadha Nope, not an issue with the logo at all. The issue is with the arrangement of the logo on the homepage.

**Comment by yashLadha** [comment]
I will try to fix-it. Currently I am outside. Soon i will send a patch.

**Comment by xeon-zolt** [comment]
naa not looking good this can be a part of about page

**Comment by meets2tarun** [comment]
@yashLadha  Are you still working on this?

**Comment by yashLadha** [comment]
i am working on it.

**Comment by meets2tarun** [comment]
@yashLadha 
If you are working and please show the progress before the PR get closed.

**Comment by yashLadha** [comment]
Send you the PR by today

File: templates/index.html [hunk 1]
```diff
@@ -14,6 +14,7 @@
         <form method="POST" enctype="multipart/form-data">
             <div class="container"><div class="row mt-3">
                 <div class="col-md-4 mx-auto">
+					<img src="https://camo.githubusercontent.com/7978821e61fc6030a9a9ea1898195ecfa82dbe15/68747470733a2f2f707265766965772e6962622e636f2f686a785033512f6d5f672e6a7067" class="img-rounded"  alt="meilix generator" width="330" height="314">
```
Comment (tabesin) [inline templates/index.html]: Why not add the image to this repo and reference it locally?

**Comment by meets2tarun** [comment]
@yashLadha Kindly check the comment request and improve it.

**Comment by yashLadha** [comment]
I will make that change

**Comment by meets2tarun** [comment]
> I will make that change

Kindly do that within a day.

**Comment by yashLadha** [comment]
Sure

**Comment by yashLadha** [comment]
@tabesin Done Review it again

**Comment by xeon-zolt** [comment]
@tabesin i think we should revert it as it do not looks good

**Comment by yashLadha** [comment]
Should I add it to about.md

**Comment by xeon-zolt** [comment]
yes it will look better in about us page which is not present currently but linked in menu

**Comment by yashLadha** [comment]
May be if it made circular them it might add something to look because
having a simple form won't look that appealing. Can u send the PR linking
that

**Comment by tabesin** [comment]
I don't think we should revert it because it solved the original issue. Rather change the code base, improve it and file a fresh PR. All the back-and-forth on the same stalled PR, then to solve merge conflicts because of its age and so on, is really not worth the effort. File a PR, I you don't like it make changes and file a new PR and yet another PR. A PR should not hang for 20 days, only then get adopted and swiftly reverted. Another argument is that I am unsure whether the orginal PR author wants to further work on it, so reversal kicks the ball into a court where we will only add frustration and it nurtures a culture of stalled code. Change often, change quickly.

**Pull request merged**